12.Hosted-Mender/docs.md

```diff
@@ -7,6 +7,81 @@ shortcode-core:
 github: false
 ---
 
+## 4.1.0-saas.2 - 2025-02-25
+
+
+### Bug Fixes
+
+
+- *(gui)* Let read-only user no longer think they could act on selected releases
+([MEN-8110](https://northerntech.atlassian.net/browse/MEN-8110)) ([dd710a7](https://github.com/mendersoftware/mender-server-enterprise/commit/dd710a723b68c15f6d92b22e3ea81356e5e638c7))  by @mzedel
+
+
+  also aligned component label + usage with reality
+  Ticket: MEN-8110
```

Comment:
This changelog entry looks malformed. The ticket is present in the body and as a formatted link. :monocle_face:

Reply:
FYI @mzedel, the `Ticket` is not parsed as part of the trailer for this one.
```diff
commit dd710a723b68c15f6d92b22e3ea81356e5e638c7
Author: Manuel Zedel <manuel.zedel@northern.tech>
Date:   Thu Feb 20 21:46:19 2025 +0100

    fix(gui): let read-only user no longer think they could act on selec
ted releases

    also aligned component label + usage with reality
+
    Ticket: MEN-8110
-
    Signed-off-by: Manuel Zedel <manuel.zedel@northern.tech>
```

Reply:
Note: this is coming straight from https://github.com/mendersoftware/mender-server-enterprise/pull/353/files#diff-84abb208895809df8ea9c2792d53b9d530a72a5c74f0cd53ea9aad41335fb761R13

Reply:
Amended here, and fixed upstream: https://github.com/mendersoftware/mender-server-enterprise/pull/366